services/network_config_manager/src/main/resources/application.properties

```diff
@@ -24,6 +24,9 @@ dataplane.isovs=true
 grpc.min-threads = 100
 grpc.max-threads = 200
 grpc.threads-pool-name = grpc-thread-pool
+grpc.number-of-channels-per-host = 10
+grpc.number-of-warmups-per-channel = 1
+grpc.monitor-hosts = 0.0.0.0,0.0.0.1
```

Comment:
@zzxgzgz if we add a new compute node, do we have to update this configuration? If so, this is basically all the nodes this NCM instance needs to monitor. 

We could leverage the Node Cache inside NCM, which has the requested information already.

Reply:
Didn't know that we have a node cache in NCM. 

However, in my tests, I am only monitoring one host, the different IP addresses is just the different interfaces of the same host. I think it is easy for us to edit.